features/conference/components/Conference.js

```diff
@@ -51,7 +68,15 @@ class Conference extends Component {
      * @returns {void}
      */
     _onPress() {
-        this.setState({ isToolbarVisible: !this.state.isToolbarVisible });
+        var isVisible = !this.state.isToolbarVisible;
+
+        this.setState({ isToolbarVisible: isVisible });
+
+        if (isVisible)
+            this.toolbarTimeout = setTimeout(
+                this._onPress,
+                toolbarTimeoutValue
+            );
```

Comment:
Absolutely. I will polish it tomorrow. It cannot be merged in this shape anyway.